02/app.js

```diff
@@ -11,7 +11,7 @@ for (i = 1; i <= 9; i++) {
 /* rozwiązanie z pętlą while  */
 
 const a = prompt('Podaj liczbę!');
-const n = 4;
+let n = 5;
```

Comment:
Nie chodziło o taką zmianę ;)
Powinieneś dodawać ilość `a` wew. pętli `while` - czego nie robisz, a masz na sztywno wpisaną liczbę.
To znacząco upraszcza zadanie ;)

![image](https://github.com/devmentor-pl/practice-js-basics/assets/55095257/ef1c02c0-c949-4b4c-8ab3-4463a9a7288e)